unicodetools/src/main/java/org/unicode/props/UnicodePropertySymbolTable.java

```diff
@@ -202,7 +202,9 @@ public boolean applyPropertyAlias0(
                     set =
                             prop.getSet(
                                     new ComparisonMatcher(
-                                            propertyValue, Relation.geq, DOUBLE_STRING_COMPARATOR));
+                                            propertyValue,
+                                            Relation.geq,
+                                            VERSION_STRING_COMPARATOR));
```

Comment:
performance issue: the propertyValue String (too bad we still always store strings...) is parsed in each comparison. would it be reasonably easy to make a variant of a ComparisonMatcher -- a VersionMatcher -- which parses the input propertyValue only once on construction, and parses each other value before comparing the VersionInfo's?

we could use a special, static final VersionInfo with, say, 255.255.255.255 for when a string is not parsable.
and 0.0.0.0 for null strings.